src/ringmesh/basic/plugin_manager.cpp

```diff
@@ -49,9 +51,47 @@
 namespace
 {
     std::vector< std::string > plugins;
+
+    bool read_plugins_configuration_file( const std::string& configuration_file )
+    {
+        try
+        {
+            GEO::LineInput file( configuration_file );
+            ringmesh_assert( file.OK() );
+            while( !file.eof() && file.get_line() )
+            {
+                file.get_fields();
+                ringmesh_assert( file.nb_fields() == 1 );
+                if( !RINGMesh::PluginManager::load_plugin( file.field( 0 ) ) )
+                {
+                    RINGMesh::Logger::err( "Plugin", "Failed to load ", file.field( 0 ) );
+                    return false;
+                }
+            }
+        }
+        catch( const std::logic_error& ex )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch a more specific exception instead of a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1181)